modules/installation-azure-day2-operations-diskencryptionsets.adoc

```diff
@@ -0,0 +1,57 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/enabling-disk-encryption-sets-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-disk-encryption-sets-day2-operator"]
+= Preparing an Azure Disk Encryption Set for Day2 operator
+The {product-title} installer can use an existing Disk Encryption Set with a user-managed key. To enable this feature, you can create a Disk Encryption Set in Azure and provide the key to the installer. 
+
+.Prerequisite
+
+To enable encryption at host by using system managed key during the installation, set the following fields in `install-config.yaml`:
+
+* `compute.platform.azure.encryptionAtHost`
+* `controlPlane.platform.azure.encryptionAtHost`
+* `platform.azure.defaultMachinePlatform.encryptionAtHost`
+
+.Procedure
+
+. Mark the node from the `encyptionATHost` cluster resource group as unschedulable using the following command:
++
+[source,terminal]
+----
+$ oc adm cordon <node_name>
+----
```

Comment:
When enabling `encyptionATHost` on worker nodes, it's better to add another step to drain node and evacuate the pods from the compute node before de-allocating vms.

Here is [similar operations to update vm property on vSphere platform](https://docs.openshift.com/container-platform/4.15/updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.html), the steps (operation on cluster) should be the same except shutting down vm/update property/starting vm, you can reference it.

I followed most steps of that doc and verified updating `encyptionATHost` on day2, cluster was running well after the changes.

Reply:
Ack